reference/secrets.html.markerb

```diff
@@ -15,37 +15,36 @@ Secrets are stored in an encrypted vault. When you set a secret through flyctl,
 
 When we launch a Machine for your app, we issue a temporary auth token to the host it runs on. The Fly.io agent on the host uses this token to decrypt your app secrets and inject them into your Machine as environment variables at boot time. When you destroy your Machines, the host environment no longer has access to your app secrets.
 
-<section class="warning">`flyctl` and our API servers are designed to prevent user secrets from being extracted. However, secrets are available to your application code as environment variables. People with deploy access _can_ deploy code that reads secret values and prints them to logs, or writes them to unencrypted data stores.</section>
+<section class="warning icon">
+**Warning:** `flyctl` and our API servers are designed to prevent user secrets from being extracted. However, secrets are available to your application code as environment variables. People with deploy access _can_ deploy code that reads secret values and prints them to logs, or writes them to unencrypted data stores.
+</section>
 
-## Setting secrets
+## Set secrets
 
 The `fly secrets set` command sets one or more app secrets, then updates each Machine belonging to that Fly App. This involves a restart of the Machine and a consequent reset of its ephemeral file system.
 
-<%= partial "docs/partials/set_secrets" %>
+The following example sets a secret that's available as the `DATABASE_URL` environment variable within your application processes:
 
-In the above example, the secret is avaliable as the `DATABASE_URL` environment variable within your application processes.
+<%= partial "docs/partials/set_secrets" %>
 
-To set, or update, a secret in the app's vault, but defer updating the Machines to later, use the `--stage` option:
+To set, or update, a secret in the app's vault, but defer updating the Machines to later, use the `--stage` option. For example:
 
 ```cmd
 fly secrets set DATABASE_URL=postgres://example.com/mydb --stage
 ```
 
-In this case, the staged secret will be available only on Machines that are started or updated after the `fly secrets set` command was run.
+In the preceding example, the staged secret is available only on Machines that are started or updated after running the `fly secrets set` command.
```

Comment:
This says the Machines ran the `fly secrets set` command, doesn't it?

Reply:
oops yeah